src/drafts/MarkdownViewer/_useListInteraction.ts

```diff
@@ -4,6 +4,17 @@ import {ListItem, listItemToString, parseListItem} from '../MarkdownEditor/_useL
 
 type TaskListItem = ListItem & {taskBox: '[ ]' | '[x]'}
 
+// Make check for code fences more robust per spec: https://github.github.com/gfm/#fenced-code-blocks
+const parseCodeFenceBegin = (line: string) => {
+  const match = line.match(/^ {0,3}(`{3,}|~{3,})[^`]*$/)
+  return match ? match[1] : null
+}
+
+const isCodeFenceEnd = (line: string, fence: string) => {
+  const match = line.match(new RegExp(`^ {0,3}${fence}${fence[0]}* *$`))
```

Comment:
```suggestion
  const match = line.match(new RegExp(`^ {0,3}${fence} *$`))
```
I think the closing fence needs to have exactly the same number of delimiters as the opening fence. For example, this is valid:

`````md
```one
````
we are still in block `one` here
```
`````

Reply:
Oh wow, you are totally right 👍